feat(Dashboards): Added a page for the pre-built dashboards (#12433)
newrelic/docs-website: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

---
title: Pre-built dasboards
tags:
  - Query your data
  - Explore and query data
  - Dashboards
  - Pre-built dashboards
---

import dashboardsDashboardIndex from 'images/dashboards_screenshot-full_dashboard-index.png'

import dashboardsPrebuiltViewAll from 'images/dashboards_screenshot-full_prebuilt-view-all.png'

import dashboardsPrebuiltSearch from 'images/dashboards_screenshot-full_prebuilt-search.png'

> Through the New Relic UI, you can access a collection of pre-built dashboards based on high levels of user interest and activity. These pre-built dashboards enhance quickstart dashboard discovery, speed up the installation process, and get to instantly visualize your data.

Reviewer: Please consider rephrasing to:
`Through the New Relic UI, you can access a collection of commonly used pre-built dashboards, to help you instantly visualize your data.`

Author: Much better, thanks!

> 

Reviewer: @nbaenam this is looking much better! Two notes:

1. "commonly pre-built" -> "commonly used pre-built" OR "popular pre-built"
2.  The second sentence is not really necessary.

Reviewer 2: Agreed- maybe something like:

> Through the New Relic UI, you can access a collection of popular pre-built dashboards, to help you instantly visualize your data for common use cases.

Author: Done!

Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the dashboards page.

<img
  title="Dashboards index"
  alt="Dashboards index"
  src={dashboardsDashboardIndex}
/>

<figcaption>
  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index.
</figcaption>

## How to install pre-built dashboards [#use]

To install a pre-built dashboard:

1. From the dashboards page, search for the pre-built dashboard you want to customize.

> 2. Click the pre-built dashboard for selecting it.

Reviewer: for selecting it -> to select it

> 3. Select the account for adding the dashboard and data you select.

Reviewer: Please consider rephrasing to:
`Select the account in which to add the data and dashboard.`

> 4. Follow the instructions you in the **Setup data source** section. You can skip this step.

Reviewer 2: "You can skip this step" --> "If you have already instrumented the data source, you can skip this step."

> 4. Follow the instructions you in the **Setup data source** section. You can skip this step.

Reviewer 2: ```suggestion
4. Follow the instructions you in the **Setup data source** section. If you have already instrumented the data source, you can skip this step.
```

Author: Done!

> 4. Follow the instructions you have in the **Setup data source** section. You can skip this step.

Reviewer: Please remove "you have", so that the first sentence becomes:

`Follow the instructions in the **Setup data source** section.`

Reviewer: What determines whether the user can skip this step?

Author: I'm not sure.. I added it because I saw it on the UI.

> 5. Click **View dashboard** to start visualizing your data. You can customize your pre-built dashboard, see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/). You can also change the name and settings, add new content and new widgets, refine the charts, etc.

Reviewer 2: ```suggestion
5. Click **View dashboard** to deploy the dashboard and start visualizing your data. Note that you can customize your new pre-built dashboard (see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/)). You can also change the name and settings, add new content and new widgets, refine the charts, etc.
```

> 5. Click **View dashboard** to start visualizing your data. You can modify your pre-built dashboard, see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/). You can change the name, the settings, add new content, add new widgets, refine the charts, etc.

Reviewer: 1. modify your pre-built dashboard -> customize the pre-built dashboard
2.  "You can change the name, the settings, add new content, add new widgets, refine the charts, etc."  ->`You can also change the name and settings, add new content and new widgets, refine the charts, etc.`

## How to view more templates [#more-templates]

> You can browse our full dashboard template catalog if you don't find the dashboard template you need. Click the **View All** link next to the Popular pre-built dashboards title to see and search for all the available pre-built dashboards. Or you can use these buttons <Icon name="fe-edit-2"/> and <Icon name="fe-edit-2"/> to pass through the last edited pre-built dashboards.

Reviewer: You can browse our full dashboard template catalog if you don't find the dashboard template you need. -> Browse the full dashboard catalog to find dashboard templates for all the available integrations.

Reviewer: to pass through the last edited -> to see the most recent

<img
  title="Search for a pre-built dashboard"
  alt="Search for a pre-built dashboard"
  src={dashboardsPrebuiltSearch}
/>

<figcaption>

>   Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index. Search for the pre-built dashboard you want to customize.

Reviewer 2: ```suggestion
  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index. Browse popular pre-built dashboards you want to install using the "<" and ">" buttons, or select the **View All** link to see the entire catalog. 
```

</figcaption>

> You can find the most popular pre-built dashboards on the left pane. To quickly find other pre-built dashboards, you can use the tabs labelled: datasources, dashboards, alerts, and app & visualizations.

Reviewer 2: I'm not sure what the left pane refers to here.

Reviewer 2: ```suggestion
You will be brought to **Add data**, where you can find hundreds of pre-built dashboards. You can use the tabs labelled: datasources, dashboards, alerts, and app & visualizations.
```

> You've a helpful left pane where the pre-built dashboards are ordered into the most popular and highest performing data sources. There are also these tabs to help you the search of a pre-built dashboard: datasources, dashboards, alerts, and app & visualizations.

Reviewer: "You've a helpful left pane where the pre-built dashboards are ordered into the most popular and highest performing data sources." ->
`You can find the most popular pre-built dashboards on the left pane.`

Reviewer: "There are also these tabs to help you the search of a pre-built dashboard: datasources, dashboards, alerts, and app & visualizations." ->

`To quickly find other pre-built dashboards, you can use the tabs labelled: datasources, dashboards, alerts, and app & visualizations.`

However, are we *sure* that all those tabs have pre-built dashboards within them?  (Please verify with Becky Darom)

<img
  title="Pre-built dashboards page"
  alt="Pre-built dashboards page"
  src={dashboardsPrebuiltViewAll}
/>

<figcaption>

>   Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index and click the **View All** link next to the Popular pre-built dashboards title.

Reviewer 2: ```suggestion
  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** > **View All** to see all pre-built dashboards in [**Add Data**](https://one.newrelic.com/marketplace).
```
